package.json

```diff
@@ -33,9 +33,9 @@
   "dependencies": {
     "@types/estree": "^0.0.45",
     "acorn": "^8.0.5",
-    "estree-walker": "^0.9.0",
-    "is-reference": "^2.0.0",
-    "periscopic": "^2.0.1",
+    "estree-walker": "^3.0.0",
+    "is-reference": "^3.0.0",
+    "periscopic": "^3.0.2",
```

Comment:
perhaps this should be 3.0.3 now. Rich just released https://github.com/Rich-Harris/periscopic/pull/12 with your fix in it today

Reply:
make that 3.0.4